Handlers/ReceiptsHandler.fs

```diff
@@ -0,0 +1,21 @@
+namespace ExpenseReports.Handlers
+
+open System.IO
+open Microsoft.AspNetCore.Http
+open Giraffe.HttpHandlers
+open Giraffe.Tasks
+
+module ReceiptHandler =
+    let receiptUploadHandler =
+        fun (next : HttpFunc) (ctx : HttpContext) ->
+            task {
+                return!
+                    (match ctx.Request.HasFormContentType with
+                    | false -> setStatusCode 400 >=> text "Bad request"
+                    | true ->
+                        for file in ctx.Request.Form.Files do
+                            let filePath = sprintf "receipts/%s" file.FileName
+                            use stream = new FileStream(filePath, FileMode.Create)
+                            file.CopyTo(stream)
```

Comment:
Would be nice if we could DI this function so that it would be more easily replaced with something like an S3 upload or whatever Azure calls their buckets if we would go with that.